.github/pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+<!--- When creating a new PR, make sure to request a review and tag related issues -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## How Has This Been Tested?
+<!--- Please run the following command on your PR to ensure code formatting consistency: -->
+<!--- autopep8 <changed files/folders> --recursive --in-place --ignore=E5 -->
+
+<!--- Please describe how you tested your changes. -->
+
+
+<!--- For new functionality, describe added unit tests. -->
+
+
+<!--- If this PR breaks existing unit tests, please describe in detail -->
+<!--- which tests break and why. Describe what functionality is changed. -->
+
+
+## Screenshots (if appropriate):
+
+## Type of change
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
+- [ ] Have you written new tests for your core changes, as applicable?
+- [ ] Have you successfully ran tests with your changes locally?
+- [ ] My code is consistent with the code style of this project.
+- [ ] My change requires a change to the documentation.
+- [ ] I have updated the documentation accordingly.
```

Comment:
Could be combined into one point

Reply:
@Askaholic I think having two points is actually beneficial here, as one could easily imagine situations where one or the other point would apply.